src/App.jsx

```diff
@@ -1,14 +1,51 @@
-import { useState } from 'react'
-import './App.css'
+import { useEffect, useState } from "react";
+import MainPage from "./Components/MainPage";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./Components/Header";
+import ArticlePage from "./Components/articlesPage";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([]);
 
-  return (
-    <div>
+  useEffect(() => {
+    const fetchArticles = () => {
+      fetch("https://crowleysnewsapi.onrender.com/api/articles?limit=200")
+        .then((resonse) => resonse.json())
+        .then((data) => {
+          setArticles(data);
```

Comment:
Please also add isLoading functionality so that users know that the data is being fetched